Automattic/wp-calypso
# Add Woo rebrand signup page

Related to #96946

## Proposed Changes

* Adds `woocommerce/rebrand-2-0` feature flag
* Adds CSS modifications for Woo rebrand 2.0
* Minor copy change (not under feature flag)

## Testing Instructions

* In your [`development.json`](https://github.com/Automattic/wp-calypso/blob/c470b69f28f940f90221730605717d06622639c7/config/development.json#L251), set `woocommerce/rebrand-2-0` to `true`
* Run `yarn start`
* In an incognito window, go to [this URL](http://calypso.localhost:3000/start/wpcc/oauth2-user?ref=oauth2&oauth2_redirect=https%3A%2F%2Fpublic-api.wordpress.com%2Foauth2%2Fauthorize%2F%3Fresponse_type%3Dcode%26client_id%3D50916%26state%3D%26redirect_uri%3Dhttps%253A%252F%252Fwoocommerce.com%252Fwc-api%252Fwpcom-signin%253Fnext%253D%25252Fwoocommerce%25252F%26blog_id%3D0%26wpcom_connect%3D1%26wccom-from%26calypso_env%3Dproduction%26client_source%26jetpack-code%26jetpack-user-id%3D0%26action%3Doauth2-login%26from-calypso%3D1&oauth2_client_id=50916)
* Observe the changes match with Figma tp2u3Nk9arcpeNMZcylNG4-fi-271_8254

#### Optional

* Set the feature flag to false
* Re-run `yarn start`
* In an incognito window, go to [this URL](http://calypso.localhost:3000/start/wpcc/oauth2-user?ref=oauth2&oauth2_redirect=https%3A%2F%2Fpublic-api.wordpress.com%2Foauth2%2Fauthorize%2F%3Fresponse_type%3Dcode%26client_id%3D50916%26state%3D%26redirect_uri%3Dhttps%253A%252F%252Fwoocommerce.com%252Fwc-api%252Fwpcom-signin%253Fnext%253D%25252Fwoocommerce%25252F%26blog_id%3D0%26wpcom_connect%3D1%26wccom-from%26calypso_env%3Dproduction%26client_source%26jetpack-code%26jetpack-user-id%3D0%26action%3Doauth2-login%26from-calypso%3D1&oauth2_client_id=50916)
* Observe the page is not affected by branding changes

#### After:

<img width="1438" alt="image" src="https://github.com/user-attachments/assets/7dd416b5-277e-4905-99d4-3e5c4f0fb841">

#### Figma:

<img width="1436" alt="image" src="https://github.com/user-attachments/assets/3baeebfc-d734-44b1-b596-418b4772ebbc">


## Pre-merge Checklist
